standard/lexical-structure.md

```diff
@@ -324,64 +340,94 @@ Multiple translations are not performed. For instance, the string literal `"\u00
 The rules for identifiers given in this subclause correspond exactly to those recommended by the Unicode Standard Annex 15 except that underscore is allowed as an initial character (as is traditional in the C programming language), Unicode escape sequences are permitted in identifiers, and the "`@`" character is allowed as a prefix to enable keywords to be used as identifiers.
 
 ```ANTLR
-Identifier
+identifier
```

Comment:
This part of the grammar has been re-formulated noting (a) the impact of the implicit rules generated from literal strings in the grammar and (b) to leverage those so "contextual keywords" are recognised *by the grammar* as identifiers when not being contextually – there might be cases when a contextual keyword is parsed as an identifier even though its in a contextual position but that can easily be handled by the compiler.